src/js/rise-cache.js

```diff
@@ -90,7 +84,7 @@ RiseVision.Common.RiseCache = (function () {
           url: url
         };
 
-      if (_isCacheRunning) {
+      if (_isCacheRunning || _isV2Running) {
```

Comment:
@donnapep I think this should remain just a check of `if (_isCacheRunning)` because `_isCacheRunning` will only be true if a 200 response code was received, whether it be v2 or not. If `_isCacheRunning` is `false` we shouldn't make the request.